### PR TITLE
add new template show port-security interface

### PR DESCRIPTION
**Pull request by jeffkala**
##### ISSUE TYPE

 - New Template Pull Request

##### COMPONENT

cisco_ios_show_port-security_interface.textfsm, cisco_ios, show port-security interface `interface`

##### SUMMARY

New template to parse the output of show port-security interface command.

```

```

### CODE REVIEW

**Comment by jmcgill298** [comment]
@jeffkala Unfortunately this PR wasn't passing tests last night when we migrated to poetry. Poetry does not allow us to package files outside of the package directory, so `templates` have moved to `ntc-templates/templates`. This branch will need to be rebased against master to fix the conflicts

**Comment by jeffkala** [comment]
No problem @jmcgill298 , I have a few fixes to the template I need to do anyways.

File: ntc_templates/templates/index [hunk 1]
```diff
@@ -174,6 +174,7 @@ cisco_ios_show_authentication_sessions.textfsm, .*, cisco_ios, show authen[[tica
 cisco_ios_show_environment_temperature.textfsm, .*, cisco_ios, sh[[ow]] envi[[ronment]] t[[emperature]]
 cisco_ios_show_ip_ospf_database_router.textfsm, .*, cisco_ios, sh[[ow]] ip ospf data[[base]] r[[outer]]
 cisco_ios_show_ip_ospf_interface_brief.textfsm, .*, cisco_ios, sh[[ow]] ip ospf int[[erface]]
+cisco_ios_show_port-security_interface.textfsm, .*, cisco_ios, sh[[ow]] por[[t-security]] i[[nterface]](\s+\S+)
```
Comment (FragmentedPacket) [inline ntc_templates/templates/index:177]:
```suggestion
cisco_ios_show_port-security_interface.textfsm, .*, cisco_ios, sh[[ow]] por[[t-security]] i[[nterface]]
```

**Review by FragmentedPacket — CHANGES_REQUESTED** [review]
Just a small change for it to work to the index file and then revert the permissions on the `development_script.py`

**Comment by jeffkala** [comment]
@FragmentedPacket , I did accept your suggestion, but not sure its accurate for my case.  Obviously I'm haivng some issues figuring it out myself.  The actual command would be `show port-security interface <interface name>`.  Last week when I talked to @jmcgill298 he helped me fix my test where I needed to change my folder name to `.._interface_interface`.  So i'm pretty confident my index file for this one needs to end with a regex to match the actual interface name?

**Comment by FragmentedPacket** [comment]
That's fair. I'm not entirely sure, but when I messed with it, I couldn't get it working with the way it was originally setup with the RegEx in the index file. You can revert it and maybe hit up @jmcgill298 to see the way to do it. I'd be interested to see how to get it working as well.

**Comment by jeffkala** [comment]
I put it back to the last regex I got working locally. using `.*` at the end.  I this is not the standard or recommended I can continue playing with it further.

**Comment by FragmentedPacket** [comment]
👍 Awesome! That makes more sense now that the template has `_interface` at the end of it.

Do you mind just doing `chmod 755 development_scripts.py` to set it back to the original permissions?

**Comment by jeffkala** [comment]
> 👍 Awesome! That makes more sense now that the template has `_interface` at the end of it.
> 
> Do you mind just doing `chmod 755 development_scripts.py` to set it back to the original permissions?

Done, sorry about that.

**Review by FragmentedPacket — APPROVED** [review]

**Review by jmcgill298 — APPROVED** [review]

**Comment by jmcgill298** [comment]
Thanks @jeffkala

**Pull request merged**